features/all-positions/PositionActionsDialog.tsx

```diff
@@ -519,8 +527,8 @@ const PositionActionsDialog = (props: DialogProps) => {
                             ? "increase"
                             : "decrease"}{" "}
                           by {Math.abs(underCollateralizedPercent).toFixed(4)}%
-                          from {latestPrice.toFixed(4)} to{" "}
-                          {underCollateralizedPrice.toFixed(4)}. You can still
+                          from {latestPrice.toFixed(8)} to{" "}
```

Comment:
why did you make this change?

Reply:
Because for example the price fo USDBTC is 0.00008X which would just be 0.0004 if only 4 deciamls. Admittedly hacky but we should fix better in future PR